variety/plugins/builtin/downloaders/EarthviewDownloader.py

```diff
@@ -54,27 +58,27 @@ def get_source_location(self):
         return self.ROOT_URL
 
     def fill_queue(self):
-        queue = Util.fetch_json(DATA_URL)
-        random.shuffle(queue)
-        return queue
+        random.shuffle(self.PHOTO_IDS)
+        return self.PHOTO_IDS
 
     def get_default_throttling(self):
         # throttle this source, as otherwise maps "overpower" all other types of images
         # with Variety's default settings, and we have no other way to control source "weights"
         return Throttling(max_downloads_per_hour=20, max_queue_fills_per_hour=None)
 
     def download_queue_item(self, item):
-        item = Util.fetch_json("https://earthview.withgoogle.com/_api/" + item["slug"] + ".json")
-        region = item["region"]
-        filename = "{}{} (ID-{}).jpg".format(
-            region + ", " if region and region != "-" else "", item["country"], item["id"]
-        )
-        origin_url = EarthviewDownloader.ROOT_URL + str(item["slug"])
```

Comment:
It's necessary to have an "identifying" `origin_url`, even if it's not "perfect" or it doesn't point to a sensible location. That's what Variety uses to identify the image, e.g. so that it does not get downloaded again if the user chooses to move it to Trash.
`origin_url` goes into `sourceURL` in the image exif metadata, which then gets used in a lot of places. E.g:
https://github.com/varietywalls/variety/blob/f3d2e7c6c89cf981994000c89f9b38823babe9ee/variety/VarietyWindow.py#L754